cv32e40s/tests/programs/custom/pmp/DefaultFull.c

```diff
@@ -38,12 +38,11 @@ void default_full()
 
   if (temp[0] == 13)
   {
-    printf("\n\t DefaultFull test pass ");
-    printf("\n\t --------------------------------------------- \n");
+    printf("\tDefaultFull pass\n");
```

Comment:
I couldn't read what the test was doing so I tweaked the printfs. Newline at the end of the line instead of the front etc.